app/javascript/retrospring/features/announcement/close.ts

```diff
@@ -0,0 +1,5 @@
+export default function (event: Event): void {
+  const announcement = (event.target as HTMLElement).closest(".announcement") as HTMLDivElement;
+  const aId = announcement.dataset.announcementId;
```

Comment:
Variables should have a clear name. Either name it `announcementId` or just `id` (because there aren't multiple we don't need to contextualize it)